src/commtrack.js

```diff
@@ -136,10 +137,14 @@ define([
             isHashtaggable: false,
             spec: {
                 date: {
-                    visibility: 'hidden',
+                    lstring: 'Date',
+                    visibility: 'visible',
                     presence: 'optional',
-                    serialize: function () {},
-                    deserialize: function () {},
+                    widget: widgets.xPath,
+                    xpathType: 'generic',
+                    serialize: mugs.serializeXPath,
+                    deserialize: mugs.deserializeXPath,
+                    help: 'The date and time of the action, e.g., now() or today()',
```

Comment:
It currently uses `now()`, right?  Assuming that works (that server-side processing accepts datetimes), I'd recommend steering them towards `now()`, and don't mention `today()` as an option, as transaction ordering is more deterministic with datetimes